### PR TITLE
Variable tags

### PR DESCRIPTION
**Pull request by ghost**
- Add possibility to specify tags for tagging created resources
- Added additional output variables to allow extension of the ecs-cluster with extra auto scaling groups and allow reuse of policies and security groups

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -12,3 +12,13 @@ output "service_role_name" {
   description = "Created IAM service role name."
   value       = "${aws_iam_role.ecs_service.name}"
 }
+
+output "iam_instance_profile_name" {
+  description = ""
```
Comment (npalm) [inline outputs.tf]: Please add a descriptions

File: variables.tf [hunk 2]
```diff
@@ -86,3 +86,9 @@ variable "user_data" {
   description = "The user-data for the ec2 instances"
   type        = "string"
 }
+
+variable "tags" {
+  type        = "map"
+  description = "A map of tags to add to the resources"
```
Comment (npalm) [inline variables.tf]: Please add a descriptions

File: outputs.tf [hunk 3]
```diff
@@ -12,3 +12,13 @@ output "service_role_name" {
...
+  value       = "${aws_iam_instance_profile.ecs_instance.name}"
+}
+
+output "instance_sg_id" {
+  description = ""
```
Comment (npalm) [inline outputs.tf]: Please add a descriptions

File: outputs.tf [hunk 4]
```diff
@@ -12,3 +12,13 @@ output "service_role_name" {
   description = "Created IAM service role name."
   value       = "${aws_iam_role.ecs_service.name}"
 }
+
+output "iam_instance_profile_name" {
+  description = ""
+  value       = "${aws_iam_instance_profile.ecs_instance.name}"
+}
+
+output "instance_sg_id" {
```
Comment (npalm) [inline outputs.tf]: Maybe rename sg to asg

File: outputs.tf [hunk 5]
```diff
@@ -12,3 +12,13 @@ output "service_role_name" {
   description = "Created IAM service role name."
   value       = "${aws_iam_role.ecs_service.name}"
 }
+
+output "iam_instance_profile_name" {
```
Comment (npalm) [inline outputs.tf]: Would it not be better to return the arn instead of the name?

**Review by npalm — CHANGES_REQUESTED** [review]
Please see review comments. Can you update the readme file as well. I use terraform-docs to generate the variables table.

File: outputs.tf (hunk 4 above)
Comment (ghost) [reply outputs.tf]: sg refers to the security group

**Review by npalm — APPROVED** [review]

**Pull request merged**